### PR TITLE
ref(rust): Remove Python processors in favor of Rust ones

### PR DESCRIPTION
**Pull request by phacops**

### CODE REVIEW

**Review by loewenheim — APPROVED** [review]

File: tests/consumers/test_message_processors.py [hunk 1]
```diff
@@ -11,27 +11,17 @@
...
 @pytest.mark.parametrize(
     "topic,processor",
     [
         ("ingest-replay-events", ReplaysProcessor),
-        ("processed-profiles", ProfilesMessageProcessor),
-        ("profiles-call-tree", FunctionsMessageProcessor),
         ("snuba-queries", QuerylogProcessor),
-        ("snuba-spans", MetricsSummariesMessageProcessor),
-        ("snuba-spans", SpansMessageProcessor),
         ("outcomes", OutcomesProcessor),
```
Comment (untitaker) [inline tests/consumers/test_message_processors.py:25]: ok then let's remove querylog too

**Review by untitaker — APPROVED** [review]

File: rust_snuba/src/processors/functions.rs [hunk 2]
```diff
@@ -100,7 +100,6 @@ struct Function<'a> {
     http_method: Option<&'a str>,
     is_application: u8,
     materialization_version: u8,
-    module: &'a str,
```
Comment (lynnagara) [inline rust_snuba/src/processors/functions.rs:103]: how come the rust processor is changing too?

Comment (phacops) [reply rust_snuba/src/processors/functions.rs:103]: This field is unused and was picked up by the Rust processor when the Python processor wasn't ingesting it.

File: rust_snuba/src/processors/spans.rs [hunk 3]
```diff
@@ -216,21 +222,6 @@ impl TryFrom<FromSpanMessage> for Span {
             .map(|(k, v)| (k, v.value))
             .unzip();
 
-        if let Some(http_method) = from.sentry_tags.http_method {
```
Comment (phacops) [inline rust_snuba/src/processors/spans.rs:219]: These are not meant to be in `tags`, just `sentry_tags`, the Python processor was wrong.

File: rust_snuba/src/processors/spans.rs [hunk 4]
```diff
@@ -216,21 +222,6 @@ impl TryFrom<FromSpanMessage> for Span {
...
-
-        if let Some(status_code) = from.sentry_tags.status_code {
-            tag_keys.push("status_code".into());
-            tag_values.push(status_code);
-        }
-
-        if let Some(transaction_method) = from.sentry_tags.transaction_method {
-            tag_keys.push("transaction.method".into());
-            tag_values.push(transaction_method);
-        }
```
Comment (phacops) [inline rust_snuba/src/processors/spans.rs:232]: These are not meant to be in tags, just sentry_tags, the Python processor had incorrect behavior.

File: rust_snuba/src/processors/spans.rs [hunk 5]
```diff
@@ -270,6 +261,7 @@ impl TryFrom<FromSpanMessage> for Span {
             trace_id: from.trace_id,
             transaction_id: from.event_id,
             transaction_op,
+            user: from.sentry_tags.user.unwrap_or_default(),
```
Comment (phacops) [inline rust_snuba/src/processors/spans.rs:264]: Setting a `user` column value was somehow forgotten despite the column being there.

File: snuba/datasets/configuration/spans/entities/spans.yaml [hunk 6]
```diff
@@ -128,7 +128,7 @@ storages:
             from_column_table: null
             from_column_name: sentry_tags
             to_nested_col_table: null
-            to_nested_col_name: tags
+            to_nested_col_name: sentry_tags
```
Comment (phacops) [inline snuba/datasets/configuration/spans/entities/spans.yaml:131]: We were fetching from `tags` instead of `sentry_tags` for the `sentry_tags` field.

**Pull request merged**